examples/summarization/run_summarization.py

```diff
@@ -682,7 +692,10 @@ def compute_metrics(eval_preds):
     training_args.generation_num_beams = (
         data_args.num_beams if data_args.num_beams is not None else training_args.generation_num_beams
     )
-
+    if is_bart:
+        training_args.generation_max_new_tokens = (
+            model.generation_config.max_new_tokens if model.generation_config.max_new_tokens is not None else model.generation_config.max_length
+        )
```

Comment:
We can remove this after #293.

Reply:
The Default config doesn't have max_new_tokens. I couldn't find anything in #293 which adds this to config. So, we need a way to provide that.
Either as part of GaudiGenerationConfig (by adding max_new_tokens based on max_length) or the current way